data/config/__init__.py

```diff
@@ -25,6 +25,9 @@ def __init__(self, filename_base, width, height):
         self.preferred_teams = json["preferred"]["teams"]
         self.preferred_divisions = json["preferred"]["divisions"]
 
+        # Game Screen Display
+        self.show_hits_errors = json["game"]["show_hits_errors"]
```

Comment:
How does this look on 32x32 boards? We could manually override this to be false on those dimensions if needed.

The way I'm handling batters due up and the at bat/pitcher names ATM is just drawing them outside the dimensions of the board, but that's obviously not ideal and I should reconsider at some point.

Reply:
Okay it looks fine unless it any of the numbers hit double digits. We can probably let people disable it in their own config then.

Reply:
I did test this, and came to the conclusion that there's just not enough space when the numbers get high.  But it does work, and because this is optional there's no harm done.